stepactions/eaas-get-ephemeral-cluster-credentials/0.1/eaas-get-ephemeral-cluster-credentials.yaml

```diff
@@ -56,3 +68,20 @@ spec:
     "${OC[@]}" get secret $SECRET -o go-template --template="{{.data.kubeconfig|base64decode}}" > $CLUSTER_KUBECONFIG
     echo "Wrote kubeconfig to $CLUSTER_KUBECONFIG"
     echo -n $(basename $CLUSTER_KUBECONFIG) > $(step.results.kubeconfig.path)
+
+    # Get username and password
+    ADMIN_PASSWORD_SECRET=$("${OC[@]}" get cti $CLUSTER_NAME -o=jsonpath='{.status.adminPassword.name}')
+    echo "Found admin password secret: $ADMIN_PASSWORD_SECRET"
+    "${OC[@]}" get secret $ADMIN_PASSWORD_SECRET -o jsonpath='{.data.username}' | base64 -d > $CLUSTER_USERNAME
+    "${OC[@]}" get secret $ADMIN_PASSWORD_SECRET -o jsonpath='{.data.password}' | base64 -d > $CLUSTER_PASSWORD
+    echo "Wrote username to $CLUSTER_USERNAME"
+    echo "Wrote password to $CLUSTER_PASSWORD"
+    echo -n $(basename $CLUSTER_USERNAME) > $(step.results.usernamePath.path)
+    echo -n $(basename $CLUSTER_PASSWORD) > $(step.results.passwordPath.path)
+    
+    # Get `APIserverURL` and `ConsoleURL` status field values 
+    API_SERVER_URL=$("${OC[@]}" get cti $CLUSTER_NAME -o=jsonpath='{.status.apiServerURL}')
+    CONSOLE_URL=$("${OC[@]}" get cti $CLUSTER_NAME -o=jsonpath='{.status.consoleURL}')
+    echo -n "$API_SERVER_URL" > $(step.results.apiServerURL.path)
+    echo -n "$CONSOLE_URL" > $(step.results.consoleURL.path)
```

Comment:
It would be nice to see these URLs output to the console as well.

Reply:
updated